details.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Intensio</title>
+    <link rel="icon" href="res/logo.svg" type="image/svg">
+</head>
+<body>
+    <nav>
+        <img src="res/logo.svg" alt="">
+        <h3 class="logo">intensio</h3>
+    </nav>
+    <main class="project-details">
+        <h1>Project name</h1>
+        <section>
+            <h2>Today</h2>
+            <div>
+                <p>20:00 - 20:15</p>
```

Comment:
Use `<time>` for such elements.

Reply:
Thanks for the review! I'm good with an 8. 

I removed the html files from here so now I'm merging this PR into `master`.